x-pack/plugins/ml/public/application/util/get_selected_ids_url.ts

```diff
@@ -7,10 +7,19 @@ import rison from 'rison-node';
 
 import { getBasePath } from './dependency_cache';
 
-export function getJobIdUrl(tabId: string, jobId: string): string {
-  // Create url for filtering by job id for kibana management table
+export enum TAB_IDS {
+  DATA_FRAME_ANALYTICS = 'data_frame_analytics',
+  ANOMALY_DETECTION = 'jobs',
+}
+
+export function getSelectedIdsUrl(
+  tabId: string,
+  ids: string | string[],
+  isGroup: boolean = false
+): string {
+  // Create url for filtering by job id or group ids for kibana management table
   const settings = {
-    jobId,
+    [isGroup ? 'groupIds' : 'jobId']: Array.isArray(ids) ? ids : [ids],
```

Comment:
having two ternaries which check for different things could produce 4 different combinations.
i don't think this should be producing `{ jobId: ['foo'] }` or `{ groupIds: 'foo' }`
one possible solution would be to have two functions `getJobIdUrl` and `getGroupIdUrl` which call a non exported shared function called `getSelectedIdsUrl ` for all the common code below.
This would remove the need to have an optional `isGroup`  parameter.